build_tools/github_actions/amdgpu_family_matrix.py

```diff
@@ -37,6 +41,7 @@
         "linux": {
             "test-runs-on": "linux-mi325-1gpu-ossci-rocm",
             "family": "gfx94X-dcgpu",
+            "bypass_tests_for_releases": True,
```

Comment:
tests are still runnning with this? we are just bypassing the result evaluation of the tests right?